package.json

```diff
@@ -15,30 +15,36 @@
 		"type": "git",
 		"url": "git://github.com/catberry/catberry-oauth2-client.git"
 	},
+	"contributors": [
+		{
+			"name": "Denis Rechkunov",
+			"email": "d.rechkunov@flamp.ru"
```

Comment:
Where did you get this email? :) The correct one is `denis.rechkunov@gmail.com`.